docs/deployment-arbitrary-cluster.md

```diff
@@ -0,0 +1,50 @@
+# Deployment on Arbitrary Cluster
+
+In this guide, we will cover the installation of NetworkManager
+and kubernetes-nmstate on your arbitrary cluster.
+
+## Requirements
+
+This guide requires you to have your own Kubernetes/OpenShift cluster. If you
+don't have one and just want to try kubernetes-nmstate out, please refer to
+the [deployment on local cluster](deployment-local-cluster.md) guide.
+
+In order to get kubernetes-nmstate running, NetworkManager
+must be installed on the node.
+
+### NetworkManager
+
+kubernetes-nmstate containers communicate with a NetworkManager instance running
+on the node using D-Bus. Make sure that NetworkManager is installed and running
+on each node.
+
+```shell
+yum install NetworkManager
+systemctl start NetworkManager
+```
+
+## kubernetes-nmstate
+
+Finally, we can install kubernetes-nmstate on our cluster.
+
+### Kubernetes
+
+```shell
+# Install k8s resources and kubernets-nmstate operator
+for manifest in "service_account.yaml role.yaml role_binding.yaml crds/nmstate_v1alpha1_nodenetworkstate_crd.yaml operator.yaml"; do
```

Comment:
should we have a release file that contains everything in one file?

Reply:
This is somethign out of the scope of the PR, we can open an issue with that.

Reply:
please just remember to create an issue  :)